packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -44,9 +48,21 @@ The third generation of the Sign-In Widget offers the same user experience as th
 
 See [Sign-In Widget, third generation](https://help.okta.com/okta_help.htm?type=oie&id=ext-compare-siw) to verify if the third generation is right for your org.
 
+### Palette generation and accessibility
+
+The third generation of the Sign-In Widget offers autogenerated palettes that preserve accessible color contrast ratios. Okta uses the [Odyssey design system](https://odyssey.okta.design/latest/foundations/accessibility/color-9O9a4XAU) to determine color contrast.
+
+Consider the following when adjusting the theming of your org:
+
+* If you don't modify your theming, your org uses the default Okta palette.
+
+* If you modify your theming using the **Theme** panel in the Admin Console, Okta generates an accessible palette. However, the generated palette might not map exactly to your brand's palette. See [Set a theme for your org](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-set-theme).
```

Comment:
I like how concise all of this is and I'm sure it's been iterated on quite a bit so none of these are hard suggestions

Did we consider adding a bit more detail about how we generate the accessible palette, for example something like "... Okta generates an accessible palette _based on the provided Primary color in the Theme panel_"?

That might be implied or understood by anyone that has done theme customizations before, but I did notice that the linked Theme doc in this line does not call this out and more detail might be better since the purpose of this update is to market gen 3's updated palette generation logic

Reply:
Good suggestion. I'll add it!